_posts/2018-03-29-Yocto-Training.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Slides from Yocto training"
+date:   2018-03-29 16:00:00 +0100
+categories: yocto training
+author: Tobias Olausson
+---
+
+We've had some new members in our PELUX team, so today I did a brief
+introduction to how the Yocto project works and what's good to keep in mind when
+using it in a PELUX context. Since the slides are still rather generic, we
+thought to share them here enjoy! And please do point out if something is wrong
+in them, nobody's perfect.
+
+As usual, the [Yocto reference
+manual](https://www.yoctoproject.org/docs/current/ref-manual/ref-manual.html) is
+the main source for everything in these slides.
+
+Download the slides [here](assets/pdf/slides-yocto-pelux.pdf), they are released
```

Comment:
Instead of making "here" the link, remove "here" and make "slides" a clickable link. Further reading: https://www.w3.org/QA/Tips/noClickHere

Reply:
Good point, done!